test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -2246,15 +2246,15 @@ Describe "Invoke-RestMethod tests" -Tags "Feature", "RequireAdminOnWindows" {
     # $dataEncodings = @("Chunked", "Compress", "Deflate", "GZip", "Identity")
     # Note: These are the supported options, but we do not have a web service to test them all.
     It "Invoke-RestMethod supports request that returns <DataEncoding>-encoded data." -TestCases @(
-        @{ DataEncoding = "gzip"}
-        @{ DataEncoding = "deflate"}
+        @{ DataEncoding = "gzip" }
+        @{ DataEncoding = "deflate" }
     ) {
         param($dataEncoding)
         $uri = Get-WebListenerUrl -Test 'Compression' -TestValue $dataEncoding
-        $result = Invoke-RestMethod -Uri $uri -ResponseHeadersVariable 'headers'
+        $result = Invoke-RestMethod -Uri $uri
 
         # Validate response content
-        $headers.'Content-Encoding'[0] | Should -BeExactly $dataEncoding
+        # If Content is compressed we cannot read it as JSON.
```

Comment:
I don't understand this comment.  Do we need to convert content from json similar to above at line 601?

Reply:
I think this comment is saying that if the content wasn't decompressed correctly then it wouldn't be valid JSON so the test would fail?  If that is case, perhaps the comment should make that more clear.

Reply:
@SteveL-MSFT  What about:
```suggestion
        # If Content is still compressed we cannot read it as JSON.
```

Reply:
The thing is, there is no `convertto-json` call following this comment, so the comment doesn't seem to make sense.

Reply:
Comment updated.